### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by HargovindArora**

### CODE REVIEW

**Comment by HargovindArora** [comment]
@nemesisdesign I have added some terms in there. Please review the style and content and suggest any changes you want to introduce in further work. :)

File: glossary/ansible-openwisp2.rst [hunk 1]
```diff
@@ -0,0 +1,27 @@
+=====================
+Technologies Involved
+=====================
+
+- `Python<https://www.python.org/>`_
```
Comment (nemesisdesign) [inline glossary/ansible-openwisp2.rst]: please leave one space before `<` and after `Python`, same applies for the rest of the terms

File: index.rst [hunk 2]
```diff
@@ -11,6 +6,10 @@ Contents:
 .. toctree::
    :maxdepth: 2
 
+   /glossary/ansible-openwisp2
+   /Setup/dev
```
Comment (nemesisdesign) [inline index.rst]: drop this and its related file for the moment, we'll think about this later

File: index.rst [hunk 3]
```diff
@@ -11,6 +6,10 @@ Contents:
...
+   /Setup/users
```
Comment (nemesisdesign) [inline index.rst]: drop this and its related file for the moment, we'll think about this later

**Comment by HargovindArora** [comment]
@nemesisdesign I'm facing difficulty for rebasing my work after Xamfy's PR! Can you please help how to merge both glossaries?

**Pull request closed without merging**